.github/workflows/check-commit.yml

```diff
@@ -8,88 +8,68 @@ on:
   workflow_dispatch:
   push:
     branches: ["main"]
-    paths-ignore:
-      - "packages/document-drive/**"
-      - "apps/academy/**"
   pull_request:
```

Comment:
I'll need someone who is fimiliar with this stuff to help me figure out how to add it to my new workflow here